website/client/js/services/userServices.js

```diff
@@ -310,6 +310,19 @@ angular.module('habitrpg')
           });
         },
 
+        setCustomDayStart: function (dayStart) {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/set-custom-day-start',
+            data: {
+              dayStart: dayStart
+            }
+          })
+          .then(function (response) {
+            Notification.text('Day start updated. Remember to refresh');
```

Comment:
That debug function needs to say "refresh" (although "sync" would be better) because cron only happens when the user syncs, and the purpose of the debug function is to force an immediate cron.

This feature shouldn't need the browser to reload (and if it does for any reason, the reload should happen automatically because users often don't read advice).